packages/cli/src/bases/Tsconfig.ts

```diff
@@ -0,0 +1,127 @@
+import fs from "fs";
+import path from "path";
+
+export namespace Tsconfig {
+  export const create = (input: { projectPath: string }): void => {
+    fs.writeFileSync(
+      path.join(input.projectPath, "tsconfig.json"),
+      JSON.stringify(tsconfig, null, 2)
+    );
+    console.log("✅ tsconfig.json created");
+  };
+
+  const tsconfig = {
```

Comment:
if you want write comment in `tsconfig.json`, It should be written as a String, not as an Object.